index.html

```diff
@@ -3211,12 +3211,28 @@ <h2>Contexts</h2>
  identifies it. This must be a <a>String</a> and must not be
  "<code>current</code>".
 
-<p>The <dfn>web window identifier</dfn>
- is the string constant "<code>window-fcc6-11e5-b4f8-330a88ab9d7f</code>".
+<p>A <dfn data-lt="web frames">web frame</dfn>
+  is an abstraction used to identify a <a>frame</a> or <a>iframe</a>
+  when it is transported via the <a href="#protocol">protocol</a>,
+  between <a>remote</a>- and <a>local</a> ends.
```

Comment:
The hyphen here seems a bit surprising to me, but I realise this is copy/pasta from "web element" above.

Looking at the whole spec, we currently have one "remote- and local ends" and one "remote and local ends". As a matter of style, I'd rather the latter.